release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -113,6 +113,20 @@ io:println(nameText); // "DanBrown"
 
 #### Runtime
 
+##### JAVA Runtime API
+
+New runtime java API is introduced for creating distinct errors which supports hierarchical errors.
```

Comment:
```suggestion
A new runtime Java API is introduced for creating distinct errors to support hierarchical errors.
```

Reply:
@warunalakshitha, do we need the "to support hierarchical errors" part here? Can't any arbitrary error be created using this method?

Reply:
Yes. we do not need. This API should use create any arbitrary error.

Reply:
Then let's change this to "A new runtime Java API is introduced for creating distinct errors to support any arbitrary error."

Reply:
I think "A new runtime Java API is introduced for creating distinct errors"

Reply:
+1